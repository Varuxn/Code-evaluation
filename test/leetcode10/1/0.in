98 144
77 26
70 72
24 8
63 66
53 73
51 60
83 84
2 4
4 44
70 41
31 69
89 59
19 66
42 61
9 15
62 20
16 43
2 34
13 97
82 95
8 16
52 16
0 1
30 60
78 94
22 54
15 96
94 15
1 7
29 32
33 36
11 53
29 41
26 35
28 58
1 57
12 54
58 75
4 10
34 6
80 90
28 95
94 88
12 86
30 74
11 59
1 2
29 76
0 14
49 78
5 49
71 89
31 39
21 79
20 45
35 56
5 21
3 81
57 66
33 87
14 29
2 6
66 12
22 42
9 92
95 10
57 49
1 24
16 30
17 20
58 59
0 83
2 44
0 12
6 91
94 58
1 51
21 27
57 62
95 27
51 64
15 27
30 82
8 18
3 23
24 37
15 83
74 5
2 3
44 65
13 36
38 85
68 93
46 74
1 40
18 67
9 28
12 55
5 72
3 33
63 80
9 95
1 52
55 70
1 9
2 22
17 47
9 17
65 6
40 48
50 84
73 20
91 7
4 5
69 76
10 15
38 46
59 88
74 78
26 53
67 68
83 9
13 17
45 92
4 87
2 77
85 95
63 71
21 26
37 72
59 12
56 79
21 32
26 38
6 8
18 31
23 66
10 19
7 11
10 13
33 39
46 50
15 25
44 63
