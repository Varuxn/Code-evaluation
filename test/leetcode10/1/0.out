1 40 3 81 12 86 13 97 15 25 15 96 16 43 17 47 18 67 22 42 40 48 42 61 51 64 58 75 63 80 67 68 68 93 80 90 